src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,145 @@
+package subway;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.*;
+
+@DisplayName("지하철 노선 관련 기능")
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+@Sql(scripts = "/line-test.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+public class LineAcceptanceTest {
+
+    /**
+     * 초기 지하철 역 생성
+     */
+    @BeforeEach
+    void initStations() {
+        Arrays.asList("장암역", "석남역", "계양역", "송도달빛축제공원역")
+                .forEach(StationCommonApi::createStation);
```

Comment:
```suggestion
var 장암역 = StationCommonApi.createStation("장암역");
var 석남역 = StationCommonApi.createStation("석남역");
var 계양역 = StationCommonApi.createStation("계양역");
var 송도달빛축제공원역 = StationCommonApi.createStation("송도달빛축제공원역");
```

역에 대해서 LineAcceptanceTest가 가지고 있으면, 인수테스트 시나리오에서 잘 쓰이지 않을까요?